test/fixtures.yml

```diff
@@ -239,6 +239,12 @@ cyber_source:
   login: X
   password: Y
 
+# Working credentials, no need to replace
+d_local:
+  login: aeaf9bbfa1
+  trans_key: 9de3769b7e
+  secret_key: ae132899f56162a669b38dab5927862f3
```

Comment:
These are public credentials or Spreedly specific?

Reply:
We usually add them to AM, there's no reason to keep them private at least for dLocal. In any case, they're public now :)

Reply:
Okay. I thought the standard was to put a placeholder, unless they are values anyone uses when testing.